content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/head.html

```diff
@@ -22,6 +22,11 @@
     <meta data-sly-test.description="${properties['jcr:description']}" name="description" content="${description}"/>
     <meta data-sly-test.templateName="${page.templateName}" name="template" content="${templateName}"/>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <sly data-sly-use.pwa="com.adobe.cq.wcm.core.components.models.PWA" data-sly-test="${pwa.pWAEnabled}">
+        <link rel="manifest" href="/content/wknd/jcr:content/manifest.webmanifest" crossorigin="use-credentials"/>
```

Comment:
looks like hard coding is left, could you please update to use the project path?

Reply:
may be you can add the getManifestPath to PWA model and call that from here, rather than stitching the strings in this html

Reply:
Done.